test/verify/check-system-services

```diff
@@ -218,13 +236,14 @@ Unit=test.service
         b.wait_not_in_text("#services-list", "test")
         self.wait_service_state("basic.target", "active")
 
-        # Make sure that symlinked services also appear in the list
-        self.wait_service_state("reboot.target", "inactive")
-        self.wait_service_state("ctrl-alt-del.target", "inactive")
-        self.wait_service_state("runlevel6.target", "inactive")
-        self.goto_service("runlevel6.target")
-        b.wait_in_text(".service-name", "Reboot")
-        b.click("#services-page .pf-c-breadcrumb__link")
+        if not user:
+            # Make sure that symlinked services also appear in the list
+            self.wait_service_state("reboot.target", "inactive")
+            self.wait_service_state("ctrl-alt-del.target", "inactive")
+            self.wait_service_state("runlevel6.target", "inactive")
+            self.goto_service("runlevel6.target")
```

Comment:
FWIW, let's remove them. They are an ancient SysV init compatibility shim, and nobody cares about these any more.